dlt/extract/incremental/__init__.py

```diff
@@ -118,6 +118,8 @@ class Incremental(ItemTransform[TDataItem], BaseConfiguration, Generic[TCursorVa
     EMPTY: ClassVar["Incremental[Any]"] = None
     placement_affinity: ClassVar[float] = 1  # stick to end
 
+    DEFAULT_DUPLICATE_CURSOR_WARNING_THRESHOLD: ClassVar[int] = 200
```

Comment:
`Incremental` is also a dataclass. you should declare `uplicate_cursor_warning_threshold` under `lag` and this is a good place to attach default value so `DEFAULT_DUPLICATE_CURSOR_WARNING_THRESHOLD` is not longer needed.

tbh. my preference would be to not create a new field at all and just keep `DEFAULT_DUPLICATE_CURSOR_WARNING_THRESHOLD` if someone want to change that, one can do it for all instances by changing the classvar. which is IMO pretty cool. WDYT?

Reply:
Good point, changed in ca3471c